pkg/vm/builtins/layouts.go

```diff
@@ -37,7 +37,7 @@ func getPlainLayout() Layout {
 }
 
 func getStarknetWithKeccakLayout() Layout {
-	return Layout{Name: "starknet_with_keccak", RcUnits: 16, Builtins: []LayoutBuiltin{
+	return Layout{Name: "starknet_with_keccak", RcUnits: 4, Builtins: []LayoutBuiltin{
```

Comment:
why 4 Rcunits? Its 16 for plain and small no?

Reply:
from the python vm: https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/lang/instances.py#L292

Reply:
oh yeah indeed. Thats like a black box for me but lets go aha